azure-pipelines.release-vnext.yml

```diff
@@ -93,6 +93,11 @@ extends:
                 displayName: Tag prelease packages with prerelease tag
                 continueOnError: true
 
+              - script: |
```

Comment:
> or implement our custom mechanism, something like
>
> parse changefiles before publish happens ( store metadata in some temp/json
run deprecate after publish

What if we run the deprecation script before the beachball release? I'd like to avoid such gymnastics if possible. 😄

Reply:
heh, yeah I guess that will work, but we can never be certain if beachball publish actually passes. then it's up to the human factor to re-run the pipelines manually again in order to properly ship stable release. wondering if `npm deprecate` would fail if we try to deprecate already deprecated ( the case of re-run after failed beachball )